orcid-persistence/src/main/java/org/orcid/persistence/dao/impl/OrgDisambiguatedDaoImpl.java

```diff
@@ -87,6 +85,19 @@ public OrgDisambiguatedEntity findByNameCityRegionCountryAndSourceType(String na
         return query.getResultList();
     }
 
+    @Override
+    public List<OrgDisambiguatedEntity> findOrgsToGroup(int firstResult, int maxResult) {
+        TypedQuery<OrgDisambiguatedEntity> query = entityManager.createQuery("from OrgDisambiguatedEntity where source_type=:ROR or source_type=:FUNDREF or source_type=:RINGGOLD",
```

Comment:
It looks like we should include another where clause so we don't try to regroup orgs that has been already grouped.

> and status !='PART_OF_GROUP'

Does that makes sense?

Reply:
I was thinking of a scenario when we want to regroup for some and re-run the cli cleaning script for everything, as this function is solely used for it.  I guess if that situation happens we can update the status to empty and re-run the script after. Changing it.